h/views/admin/admins.py

```diff
@@ -36,7 +36,9 @@ def admins_add(request):
     user = models.User.get_by_username(request.db, username, authority)
     if user is None:
         request.session.flash(
-            _("User {username} doesn't exist.".format(username=username)), "error"
+            # pylint:disable=consider-using-f-string
+            _("User {username} doesn't exist.".format(username=username)),
+            "error",
```

Comment:
I can't imagine we're doing internationalisation + parameterisation correctly. How would a translation of this string work?